openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OParamUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Feedzai
+ * Copyright 2019 Feedzai
```

Comment:
less than nitpick (consider knowledge for your future self): I think the year of the copyright is not supposed to be changed; it should be the year of the first version of the file (we had that discussion recently at an internal project)